checks/check_kafka_connect.go

```diff
@@ -32,7 +33,7 @@ func main() {
     }
   }
 
-  hostString := args.ProtocolString + "://" + args.Host + ":" + strconv.Itoa(args.Port)
+  hostString := args.ProtocolString + "://" + args.Host + ":" + strconv.Itoa(args.Port) + args.Path
```

Comment:
Is this backwards compatible (i.e. does it throw an error if args.Path is not passed in?)

Reply:
Path is not required